src/Core/Addon/Module/ModuleManager.php

```diff
@@ -294,7 +294,9 @@ public function install($source)
         if (!empty($source)) {
             $this->moduleZipManager->storeInModulesFolder($source);
         } elseif (!$this->moduleProvider->isOnDisk($name)) {
-            $this->moduleUpdater->setModuleOnDiskFromAddons($name);
+            if (!$this->moduleUpdater->setModuleOnDiskFromAddons($name)) {
+                throw new Exception($this->translator->trans('The module %name% has not been found on Addons.', ['%name%' => $name]));
```

Comment:
```suggestion
                throw new Exception($this->translator->trans('The module %name% could not be found on Addons.', ['%name%' => $name])); 'Admin.Modules.Notification'
```

Reply:
Be careful, there's a syntax error in your suggestion.

```suggestion
                throw new Exception($this->translator->trans('The module %name% could not be found on Addons.', ['%name%' => $name], 'Admin.Modules.Notification'));
```